frontstage/templates/surveys/surveys-confirm-organisation.html

```diff
@@ -0,0 +1,31 @@
+{% import 'partials/section.html' as section %}
```

Comment:
Is this exactly the same as the other `confirm-organisation-survey.html` page? 
If it is it'd be better to reuse it rather than adding a new one

Reply:
Yes it is the same as the register page, I thought it warranted a new html page as this is for adding a survey for the future if changes needed to be made then the foundation is there. I do see your point but it depends how this screen is going to be iterated.

Reply:
If it's the exact same I think we should just reuse it

Reply:
Sorry my mistake these pages are different, they point to different places once you continue